src/App.tsx

```diff
@@ -1,19 +1,17 @@
 import React, { Component } from "react";
 import { BrowserRouter, Route, Switch } from "react-router-dom";
 
+// tslint:disable: ordered-imports
 import MainPage from "./components/mainPage/mainPage";
 import Navigation from "./components/navigationBar/navigationBar";
```

Comment:
What do you expect when you click on `Navigation`? I don't have any imagination here.

Reply:
Navigation is a class which includes the elements from the navigation bar. for example if you want to use a sidebar instead to have to modify this class. look inside ./components/navigationBar/navigationBar to see how it works or just do "npm run start" on this branch

Reply:
"npm run start" doesn't work for me. I get this error
```
./src/components/proofEditor/issueInformation.tsx
Module not found: Can't resolve 'shortid' in 'C:\Users\kweir\Desktop\Uni\SoSe19 - Vorlesungen_Arbeit\Diproche\github\webinterface\src\components\proofEditor'
```

A git pull didn't resolve this problem either. That's why I asked in the first place. Do you have another suggestion for a fix?

Reply:
Did u tried "npm install" first? the branch worked for me after doing that.

Reply:
No, I didn't. That seemed to be the problem for me, too. Thanks!